src/blurb/blurb.py

```diff
@@ -482,21 +482,24 @@ def finish_entry():
                 # we'll complain about the *first* error
                 # we see in the blurb file, which is a
                 # better user experience.
-                if key == "gh-issue" and int(value) < lowest_possible_gh_issue_number:
-                    throw(f"The gh-issue number must be {lowest_possible_gh_issue_number} or above, not a PR number.")
-
                 if key in issue_keys:
                     try:
                         int(value)
                     except (TypeError, ValueError):
                         throw(f"Invalid {issue_keys[key]} issue number! ({value!r})")
 
+                if key == "gh-issue" and int(value) < lowest_possible_gh_issue_number:
+                    throw(f"The gh-issue number must be {lowest_possible_gh_issue_number} or above, not a PR number.")
```

Comment:
```suggestion
                        throw(f"Invalid {issue_keys[key]} number: {value!r}")

                if key == "gh-issue" and int(value) < lowest_possible_gh_issue_number:
                    throw(f"Invalid gh-issue number: {value!r} (must be >= {lowest_possible_gh_issue_number})")
```

Rather than assuming the user entered a PR number (which are now high enough to be valid), let's tell them what the invalid value actually is.  I also rephrased the messages to be more similar.

On a separate note, should there be an upper bound check for bpo numbers too?

Reply:
> On a separate note, should there be an upper bound check for bpo numbers too?

Ideally we should only be using GH numbers from now on, using corresponding GH numbers for old migrated, bpo issues.

But I don't mind adding extra bpo validation, if you're volunteering a PR :)